Description: Use proper python shebang
Origin patch from Per Andersson <avtobiff@gmail.com>
Author: Alexandre Ghiti <alexandre.ghiti@canonical.com>
Origin: other
Last-Update: 2022-01-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/pygments/mentos.py
+++ b/lib/pygments/mentos.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
 import struct
